ui/raidboss/popup-text.js

```diff
@@ -256,6 +256,11 @@ class PopupText {
     if ('disabled' in trigger && trigger.disabled)
       return;
 
+    // If using named groups, treat matches.groups as matches
+    // so triggers can do things like matches.targetName.
```

Comment:
Insanely minor, but the o5n example has `matches.target` not `targetName`

Reply:
Thanks! Insanely minor feedback still appreciated.  :)